schemas/json-ld/iden3credential.json-ld

```diff
@@ -39,7 +39,11 @@
         "credentialSubject": {
           "@id": "iden3-vocab:credentialSubject",
           "@type": "@id"
-        },              
+        },
+        "subjectPosition": {
+          "@id": "iden3-vocab:subjectPosition",
+          "@type": "xsd:string"
```

Comment:
Maybe it should be bolean?

Reply:
I think for end user it will be confused.
`subjecPosition: true`. What is mean? xD

Reply:
no.subjectPosition : index is ok

Reply:
done